### DISCUSSION TITLE
iptv world cup 2022

### DISCUSSION THREAD

**Discussion by uwansm** [iptv-org/community#295] [Q&A] [answered]
anyone have iptv world cup 2022 ?

**Comment by pukozs75** [comment, marked as answer]
You can watch WC 2022 on TVCG1, TVCG2, TVCG Parlamentarni on Montenegro playlist
There is also Polish TVP1 and TVP2 and they are also broadcasting WC.

**Reply by uwansm** [reply]
thanks you
